app/assets/javascripts/discourse/app/instance-initializers/video-placeholder.js

```diff
@@ -0,0 +1,54 @@
+import { withPluginApi } from "discourse/lib/plugin-api";
+import { iconHTML } from "discourse-common/lib/icon-library";
+
+export default {
+  initialize() {
+    withPluginApi("0.8.7", (api) => {
+      function _handleEvent(event) {
+        const svgIcon = event.target;
+        const overlay = svgIcon.parentElement;
+        const wrapper = overlay.parentElement;
+        const parentDiv = wrapper.parentElement;
+
+        wrapper.style.display = "none";
```

Comment:
Maybe we remove this element for cleaner markup (and event listener removal as well)?